static/src/components/builder/store/modules/conditionalDisplay.js

```diff
@@ -9,16 +9,20 @@ const initialState = () => {
   };
 };
 
-export const state = {
-  ...initialState()
-};
+export const state = initialState();
 
+function* getErrors (state) {
+  if (state.condition.includes('"')) yield ['condition', ["Please avoid using [ \" ] and use [ ' ] instead."]];
```

Comment:
The error message shouldn't be inside an array. It should be:
`
['condition', "Please avoid using [ \" ] and use [ ' ] instead."]
`

`utils.toMultiDict` will append the error to an array for you.